class.jetpack-gutenberg.php

```diff
@@ -206,6 +206,14 @@ public static function enqueue_block_editor_assets() {
 			plugins_url( '_inc/blocks/', JETPACK__PLUGIN_FILE )
 		);
 
+		$is_subscriptions_active = Jetpack::is_module_active( 'subscriptions' );
+		if ( $is_subscriptions_active ) {
+			// Get number of subscribers
+			$subscriptions = new Jetpack_Subscriptions_Widget();
```

Comment:
I'm getting the following error when the Subscriptions module is not active on the site:

`Fatal error: Class 'Jetpack_Subscriptions_Widget' not found in wp-content/plugins/jetpack-dev/_inc/lib/admin-pages/class.jetpack-react-page.php on line 235`

Reply:
Hey @jeherve , can you please tell me how I can reproduce this fatal error? I’ve tried disabling the Subscriptions module and editing a post with a Subscription block in it as well as creating a new post, but neither resulted in any errors in my error log.